nix/kexec-installer/restore_routes.py

```diff
@@ -14,7 +14,7 @@ def filter_interfaces(network: list[dict[str, Any]]) -> list[dict[str, Any]]:
             continue
         addr_info = []
         has_dynamic_address = False
-        for addr in net["addr_info"]:
+        for addr in net.get("addr_info"):
```

Comment:
One cannot iterate over None:

```suggestion
        for addr in net.get("addr_info", []):
```

Reply:
ah yeah right